src/drc/api/serializers.py

```diff
@@ -128,6 +128,10 @@ class EnkelvoudigInformatieObjectSerializer(serializers.HyperlinkedModelSerializ
         help_text=_("Aanduiding van de rechtskracht van een informatieobject. Mag niet van een waarde "
                     "zijn voorzien als de `status` de waarde 'in bewerking' of 'ter vaststelling' heeft.")
     )
+    locked = serializers.SerializerMethodField()
```

Comment:
```suggestion
    locked = serializers.BooleanField(read_only=True, source="canonical.lock")
```
I *think*. I'm not sure if the field itself will do the type cast